app/models/vha_facility_adapter.rb

```diff
@@ -121,7 +121,7 @@ def self.mh_clinic_phone(attrs)
   end
 
   def self.services_date(attrs)
-    Time.at(attrs['OutpatientServicesDataDate'] / 1000).utc.to_date.iso8601 if attrs['OutpatientServicesDataDate']
+    Date.strptime(attrs['FacilityDataDate'], '%m-%d-%Y').iso8601 if attrs['FacilityDataDate']
```

Comment:
I will do a more thorough fallback case but for now, all records in GIS are populated and with the same value. We have an operational reason for wanting to get this in - we need a refresh and the key person on the GIS team is disappearing for two weeks.